src/python/pants/VERSION

```diff
@@ -1 +1 @@
-1.9.0rc0
+1.9.0rc1
```

Comment:
Please leave this unchanged. This PR would land on master, so the diff to change version to `1.9.0rc1` should land on the 1.9.x branch separately.

Reply:
Got it. I will make that change to the `1.9.x` branch, which I will create now. The change was reverted in d46f679.

Reply:
Whoops. Yep, that too. https://www.pantsbuild.org/release.html#preparing-a-release-from-a-stable-branch